io/net/netperf_test.py.data/netperf_test.yaml

```diff
@@ -10,6 +10,7 @@ EXPECTED_THROUGHPUT: 90
 duration: 120
 minimum_iterations: 1
 maximum_iterations: 5
+mtu: "1500"
```

Comment:
@smruti77 Also, run this for all mtu sizes

Reply:
and for iperf ..?

Reply:
yes please